### PR TITLE
Add missing JSONPath and sql files (close #1088)

### LINKED ISSUE #1088: Add missing JSONPath and SQL files
Pipelines are often now running automigrations so we stopped publishing jsonpath and sql files to iglu central. However, not all pipelines have automigration technology so this issue will add in the missing jsonpaths and sql files. We should also continue publishing them for the forseeable future.

### PR DESCRIPTION
**Pull request by paulboocock**
cc @ihortom for visibility.

### CODE REVIEW

**Comment by paulboocock** [comment]
I've cut the scope of this back and removed the sauna schemas as discussed.

**Review by cksnp — APPROVED** [review]
Approving to unblock @paulboocock but I really don't know if anything is missing here so if @chuwy or @jbeemster know more it'd be great for them to confirm.

File: jsonpaths/com.google.analytics.ecommerce/item_1.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (chuwy) [inline jsonpaths/com.google.analytics.ecommerce/item_1.json:1]: @paulboocock I think you're better positioned to assess its importance. I have no idea where it is used.

File: jsonpaths/com.google.analytics.ecommerce/transaction_1.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (chuwy) [inline jsonpaths/com.google.analytics.ecommerce/transaction_1.json:1]: Same as `item_1`

File: jsonpaths/com.maxmind/anonymous_ip_1.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (chuwy) [inline jsonpaths/com.maxmind/anonymous_ip_1.json:1]: This one has JSON Schema, but no implementation. I think worth to keep.

File: jsonpaths/org.ietf/http_client_hints_1.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (chuwy) [inline jsonpaths/org.ietf/http_client_hints_1.json:1]: This is the one Support have raised, so probably the most important.

**Review by chuwy — APPROVED** [review]
LGTM!

**Comment by chuwy** [comment]
Closing in favor of https://github.com/snowplow/iglu-central/pull/1095

**Pull request closed without merging**